recipes/cerf-feedstock/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set name = "cerf" %}
+{% set version = "1.16" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://jugit.fz-juelich.de/mlz/libcerf/-/archive/v{{ version }}/libcerf-v{{ version }}.tar.gz
+  sha256: f24cfe72716e008ebe94376e2fb08cf9f6d982a430e0688e2f1af82da51e6d2d
+
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+
+test:
+  commands:
+    - test -f "$CONDA_PREFIX/lib/libcerf.so"  # [linux]
+    - test -f "$CONDA_PREFIX/lib/libcerf.dylib"  # [osx]
```

In the build process, you should use `$PREFIX` instead. Also you can unify this as:
```suggestion
    - test -f "$PREFIX/lib/libcerf${SHLIB_EXT}"  # [unix]
```

---

I didn't know about those environment variables. I'll make the changes to the test section. Did you want me to make changes to `build.sh` as well?

---

No, everything is fine now. We're only waiting for CI.